posts/2022-10-06-experimenting-with-voting-schemes.adoc

```diff
@@ -0,0 +1,400 @@
+= Introduction to e-voting schemes
+agoetschm
+v1.0, 2022-10-06
+:title: Introduction to e-voting schemes
+:lang: en
+:tags: [voting, crypto, zero-knowledge, en]
+
+Given the availability of technology in our society as well as the need for tools supporting democratic decision making, it is not surprising that e-voting is a growing research field.
+But unlike suggested by their name, e-voting schemes are not only digitalized version of traditional voting systems where paper ballots are counted by hand.
+They go a step further by leveraging cryptographic methods to provide privacy and verifiability guarantees. 
+In particular, trust is shifted from the system to independent verifiers who can check that everything went well.
+
+The aim of this post is to gain an understanding of how this type of systems works with a concrete example. We will go over some background information on voting schemes, give a description of the Helios 2.0 scheme, look into a possible attack and finish with a few pointers about key parts of a simplified implementation.
+
+
+== Types of voting schemes
+In traditional voting systems, each voter puts a piece of paper carrying her vote in a box. Some trusted administrator then opens the box, counts the votes and publishes the result.
```

Comment:
Traditional conundrum with choosing sexes for a subject. Solution is to either utilise her/his or use the proposed change.

```suggestion
In traditional voting systems, each voter puts a piece of paper carrying their vote in a box. Some trusted administrator then opens the box, counts the votes and publishes the result.
```

Reply:
I will leave it like this, a little feminism shouldn't hurt here :)